build.gradle

```diff
@@ -141,7 +141,7 @@ ext {
   userMaxTestRetryFailures = project.hasProperty('maxTestRetryFailures') ? maxTestRetryFailures.toInteger() : 0
 
   skipSigning = project.hasProperty('skipSigning') && skipSigning.toBoolean()
-  shouldSign = !skipSigning && !version.endsWith("SNAPSHOT") && !version.contains("-alpha") && project.gradle.startParameter.taskNames.any { it.contains("upload") }
+  shouldSign = !skipSigning && !version.endsWith("SNAPSHOT") && !version.contains("-alpha") && !version.matches("[0-9]*\\.[0-9]*\\.[0-9]*-[0-9]*-(ccs|ce)") && project.gradle.startParameter.taskNames.any { it.contains("upload") }
```

Comment:
This and other PRs have been going on for awhile so I might just be forgetting, but why did we need this filter? Is it just to avoid having e.g. the signing keys setup for the equivalent of snapshot builds?

Reply:
@ewencp yeah, signing was failing. i see that we don't do signing on the snapshot / product repo builds currently so i am skipping them for now on the nano version builds as well. but if we want to use these in product i will have to figure out how to get that working.